modules/bringauto_repository/GitLFSRepository.go

```
@@ -43,40 +50,155 @@
+			logger.ErrorIndent(errorPackPath)
+		}
+		return fmt.Errorf("packages in Git Lfs are not subset of packages in Json definitions")
+	}
+
+	if len(expectedPackForImagePaths) > 0 {
+		logger.Warn("Expected %d packages (built for target image) to be in git lfs (listing first %d):", len(expectedPackForImagePaths), listFileCount)
+		for i, expectedPackForImagePath := range expectedPackForImagePaths {
+			if i > listFileCount - 1 {
+				break
+			}
+			logger.WarnIndent(expectedPackForImagePath)
```

golangci-lint (1.62.2)

printf: non-constant format string in call to (*bringauto/modules/bringauto_log.Logger).WarnIndent

(govet)